src/openrct2/actions/SmallSceneryPlaceAction.hpp

```diff
@@ -49,6 +49,11 @@ DEFINE_GAME_ACTION(SmallSceneryPlaceAction, GAME_COMMAND_PLACE_SCENERY, GameActi
     {
     }
 
+    uint32_t GetCooldownTime() const override
+    {
+        return 20;
```

Comment:
I wonder if this is correct, I took the value that was previously used but this doesn't even cover a single game tick of 25ms.

Reply:
It's to block the cluster tool that issues 16+ actions in one tick.

Reply:
I see, I'm not even sure if this still works when nested actions are used since this is only checked at network level.

Reply:
Cluster isn't a nested action (yet).

Reply:
Just saying, also is this ready to be merged?